basic-pr-verification/testdata/pull_request_title_with_dot.json

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
I wonder if we could and should move that shared test data one level up, from `/<action>/testdata` to `/testdata`

Reply:
Sounds reasonable to me! Should I do the same for `detect-matching-pr` action too? Actually, I'd better reuse the push payload from there, somehow I didn't notice it.